src/main/scala/org/hammerlab/guacamole/commands/jointcaller/SomaticJointCaller.scala

```diff
@@ -66,15 +65,13 @@ object SomaticJoint {
   def inputsToReadSets(sc: SparkContext,
```

Comment:
hadn't noticed this before -- I guess `InputCollection` has escaped the joint caller and is used by top level guacamole code? I'm not necessarily against that if it is useful elsewhere, but I think in general we should probably wait until the joint caller matures more before pulling its stuff out into general infrastructure